test/Npgsql.Tests/PgPassFileTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using NUnit.Framework;
+
+namespace Npgsql.Tests
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.None)]
+    public class PgPassFileTests : TestBase
```

Comment:
If I understand correctly these are true unit tests - they only check PgPass components and don't actually initiate a connection. It's great to have these, but the testsuite shouldn't inherit from TestBase (which is meant for integration tests which do actual database interaction).

For these tests I'd bypass the environment variable and simply instantiate a PgPassFile directly with your own mock path. This reduces impact on the environment, you can remove the `[Parallelizable(ParallelScope.None)]`, etc.